rfcs/text/0000-Incident-Response-Plan.md

```diff
@@ -0,0 +1,199 @@
+This Incident Response Plan (IRP) identifies and describes the Human Cell Atlas (HCA) Data Coordination Platform (DCP) process for investigating, tracking, and responding to cybersecurity incidents. It is in place to both manage and respond to any incidents or potential incidents that may occur. The goal of this plan is to detect and react to security incidents, determine their scope and risk, respond appropriately, communicate results and risks to stakeholders, and reduce the likelihood of the incident from reoccurring.
+
+### Scope
+This document establishes processes and procedures for implementing a security incident handling capability for the DCP system, the data contained within the system, and any person who gains access to the DCP system or data. 
+
+### Definitions
+* An __event__ is any observable occurrence in a system or network.
+* An __adverse event__ is an event with a negative consequence, such as packet floods, unauthorized use of system privileges, unauthorized access to data, and execution of malware that destroys data.
+* An __incident__ is a violation or imminent threat of violation of security policies, acceptable use policies, or standard security practices that actually or potentially jeopardizes the DCP’s ability to provide services or results in the exfiltration, change, deletion, or other compromise of data the system processes, stores, or transmits. An incident may occur as a result of malicious activity or internal error (e.g., operational or developmental errors).
+* A __breach__ is any successful compromise at any level of protective controls to, or unauthorized access to or use of, the DCP system or data. Any attempt, whether successful or unsuccessful, is an incident, making a breach a subset of incidents.
+
+### Roles and Responsibilities
+The primary individuals and roles listed below play a part in the DCP incident reporting and response processes.
+* __DCP Incident Response Team__: The DCP IRT serves as the focal point for IT security incidents for the DCP. The DCP IRT identifies security incidents, characterizes the nature and severity of incidents, and provides diagnostic and corrective actions when appropriate. The IRT will receive any reports of suspicious or confirmed incidents. The DCP IRT will be comprised of DCP Security Leads and security team members. 
+* __Incident Response Team Coordinator__: The IRT Coordinator is a member of the IRT responsible for assembling all data pertinent to an incident, communicating with the appropriate parties, ensuring information is complete, overseeing the handling of the incident, and reporting on incident status both during and after investigation. This individual ensures the procedures outlined in this plan are effectively implemented. The IRT Coordinator is not intended to be a part of the solution team. Only the DCP Incident Response Team Coordinator is authorized to communicate directly with the HCA Oversight Committee for purposes of reporting an incident.
```

Comment:
Is there a separate document that outlines guidance for when the IRT Coordinator will notify folks like legal, communications, etc? Personally, I've found having that written down for folks leading response to be very helpful, especially when incidents aren't very frequent.

Reply:
There are high-level communication/escalation steps outlined in the process portion of this IRP. Per this plan, the IRT Coordinator is responsible for notifying DCP security, business stakeholders, and the Oversight Committee; legal, communications teams, etc. will be notified based on feedback from Oversight either by Oversight or as Oversight otherwise directs.